scipy/cluster/tests/test_vq.py

```diff
@@ -289,3 +289,24 @@ def test_kmeans_large_thres(self):
         res = kmeans(x, 1, thresh=1e16)
         assert_allclose(res[0], np.array([4.]))
         assert_allclose(res[1], 2.3999999999999999)
+        
```

Comment:
There is some whitespace here, and it should be removed (PEP8 W292).

Reply:
Done